airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -1503,6 +1503,8 @@ def insert_job(
         project_id: Optional[str] = None,
         location: Optional[str] = None,
         nowait: bool = False,
+        retry: Optional[Retry] = None,
```

Comment:
Just to clarify what is the reason we use `None` rather than `DEFAULT_RETRY`?
What is the default behavior or job.result() ? I assume there was some value for retry set in the underlying service regardless if we set it explicitly or not? (Just want to make sure that this is not a breaking change)

Reply:
Default value was `DEFAULT_RETRY` to keep the same behavior that we had previously I changed `None` to `DEFAULT_RETRY`